SwiftUIDZ/SwiftUIDZ/BuySubscriptionView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  BuySubscriptionView.swift
+//  SwiftUIDZ
+//
+//  Created by Анна Сычева on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Вью подписки
+struct BuySubscriptionView: View {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let spacing: CGFloat = 20
+        static let calendarImageName = "calendar"
+        static let imageSize: CGFloat = 130
+        static let fontSize: CGFloat = 50
+        static let padding: CGFloat = 30
+        static let daysText = "days"
+        static let rubText = "RUB"
+        static let buyText = "Buy"
+        static let buttonWidth: CGFloat = 100
+        static let buttonHeight: CGFloat = 50
+        static let cornerRadius: CGFloat = 8
+    }
+    
+    // MARK: - Public Properties
+    
+    let day: Int
+    let price: Int
+    
+    @EnvironmentObject var subscriptionOptions: SubscriptionOptions
```

Comment:
```suggestion
    @EnvironmentObject private var subscriptionOptions: SubscriptionOptions
```
Если не ругается, то private